src/components/ProgressBars/LinearProgressBar/__stories__/linearProgressBar.stories.mdx

```diff
@@ -0,0 +1,214 @@
+import LinearProgressBar from "../LinearProgressBar";
+import { ArgsTable, Story, Canvas, Meta } from "@storybook/addon-docs";
+import { createComponentTemplate } from "../../../../storybook/functions/create-component-story";
+import colors from "../../../../constants/colors.json";
+import DialogContentContainer from "../../../DialogContentContainer/DialogContentContainer";
+import { Info } from "../../../Icon/Icons";
+import Icon from "../../../Icon/Icon.jsx";
+import { SPINNER, STEPPER, SKELETON } from "../../../../storybook/components/related-components/component-description-map";
+import { Link } from "../../../../storybook/components";
+import BreadcrumbsBar from "../../../BreadcrumbsBar/BreadcrumbsBar.jsx";
+import BreadcrumbItem from "../../../BreadcrumbsBar/BreadcrumbItem/BreadcrumbItem.jsx";
+import "./linearProgressBar.stories.scss";
+
+<Meta
+  title="Components/LinearProgressBar"
+  component={ LinearProgressBar }
+/>
+
+<!--- Component template -->
+
+export const lineProgressBarTemplate = (args) => {
+  return (
+    <div className="linear-progress-bar_small-wrapper">
+      <LinearProgressBar value={args.value} size={args.size} />
+    </div>
+  )
+} 
+
+<!--- Component documentation -->
+
+# LinearProgressBar
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+- [Variants](#variants)
+- [Do’s and don’ts](#dos-and-donts)
+- [Use cases and examples](#use-cases-and-examples)
+- [Related components](#related-components)
+- [Feedback](#feedback)
+
+## Overview
+Progress bars show continuous progress through a process, such as a percentage value. They show how much progress is complete and how much remains.
+<Canvas>
+  <Story name="Overview"
+         args={ {value: 20, size: LinearProgressBar.sizes.LARGE} }>
+    { lineProgressBarTemplate.bind({}) }
+  </Story>
+</Canvas>
+
+## Props
+<ArgsTable of={ LinearProgressBar } />
+
+## Usage
+<UsageGuidelines guidelines={[
+  "Give users an indication of how much of the task has been completed and how much is left."
+]}/>
+
+<Tip>If you need to lead a user through a progress, use the <Link href="/?path=/docs/components-Wizard--overview" size={Link.sizes.SMALL} withoutSpacing>Wizard</Link> instead.</Tip>
+
+## Variants
+### Regular
+<Canvas>
+  <Story name="Regular">
+    <div className="linear-progress-bar_small-wrapper">
+      <div className="linear-progress-bar_block">
+        <LinearProgressBar indicateProgress value={30} size={LinearProgressBar.sizes.LARGE} />
```

Comment:
please set the progress bar width by using the className prop instead of by using a wrapper